lib/connection.js

```diff
@@ -24,12 +24,53 @@ var EndpointState = require('./endpoint.js');
 var Session = require('./session.js');
 var Transport = require('./transport.js');
 
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
 var net = require('net');
 var tls = require('tls');
 var EventEmitter = require('events').EventEmitter;
 
 var AMQP_PROTOCOL_ID = 0x00;
 
+function find_connect_config() {
+    var paths;
+    if (process.env.MESSAGING_CONNECT_FILE) {
```

Comment:
Please document this environment variable in the README

Reply:
Please provide an example in the README.md that covers most of the properties (general use case). It will make it easier for customers to use as a base reference.